### PR TITLE
Fix logo rotate behaviour

### PR DESCRIPTION
**Pull request by Caruki**

### CODE REVIEW

File: src/components/chimera.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg height="511pt" viewBox="-8 0 511 511.96139" width="511pt" xmlns="http://www.w3.org/2000/svg" style="fill: #f3cfd6;">
```
Comment (marwinburesch) [inline src/components/chimera.svg]: good old `pt`, `px` might be more appropriate 🎓

File: src/components/chimeraflipped.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<svg height="511pt" viewBox="-8 0 511 511.96139" width="511pt" xmlns="http://www.w3.org/2000/svg" style="fill: #f3cfd6; transform: scaleX(-1);">
```
Comment (marwinburesch) [inline src/components/chimeraflipped.svg]: same here as in `chimera.svg`

File: src/components/chimera.svg (hunk 1 above)
Comment (Caruki) [reply src/components/chimera.svg]: fixed 🙇‍♀

File: src/components/chimeraflipped.svg (hunk 2 above)
Comment (Caruki) [reply src/components/chimeraflipped.svg]: fixed 🙇‍♀

**Pull request merged**